src/http/RequestMethod.cc

```diff
@@ -194,7 +194,7 @@ HttpRequestMethod::isIdempotent() const
     // RFC 5789 - TODO check
     // RFC 5842 - TODO check
 
-    // RFC 7540 section 11.6
+    // RFC 9113 section 3.4
```

Comment:
FWIW, the old reference is much more useful than the new one. The old reference points to the PRI method [registration](https://www.rfc-editor.org/rfc/rfc7540#section-11.6) where this fake method idempotency is defined. The new reference points to a section about HTTP/2 magic that does not define PRI as a method at all, forcing the reader to search the new RFC and find ... the old reference!

I do not insist on any changes here, but I would keep the old reference.

```suggestion
    // RFC 7540 section 11.6
```